### PR TITLE
Adds use case scenario

### PR DESCRIPTION
**Pull request by waldemarnt**
Adds new use case scenario that shows how to filter
models by brands, series and equipment types from telemetry api.
screen shot 2016-07-27 at 3 13 49 pm

@jeffhsta and @waldemarnt

### CODE REVIEW

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 1]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
 
 More details can be found on link [Get single equipment section](#get-equipment-id)
 
+## Listing models filtering by series, types and brands
+
+To list models applying a filter by series, types and brands is necessary to make HTTP requests to [Telemetry API](#telemetry-api) resources using specific query parameters.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/is/it is

**Comment by glauco** [comment]
Our documentation is currently broken due to some changes on Swagger contracts (I am guessing). We will fix this prior accepting this PR.

Could we keep this on hold while we fix this? :)

**Comment by waldemarnt** [comment]
@glauco the problem happened on the dev environment that is unstable. In this pull request https://github.com/agco-fuse/documentation/pull/61/commits/2558b331c05055e7eda14e57ae5a262a910cc4c8 I just changed to use sandbox environment instead.

**Comment by glauco** [comment]
@waldemarnt great! Would it be possible submitting this as a separated pull request?

This seem to be a pretty straightforward fix, while reviewing the documentation, as a whole, is a little bit more complicated. What are your thoughts on it?

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 2]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
...
+```curl
+curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
+  "https://agco-fuse-trackers-sandbox.herokuapp.com/models?\
+  links.series=fbbdb6a9-1806-4ef1-9d4f-7841a472220e&\
+  links.equipmentType=fbbdb6a9-1806-4ef1-9d4f-7841a472220e"
+```
+
+To retrieve a list of models it's necessary to make a HTTP request to `models` resource on Telemetry API.
+
+Models are linked with series and equipment type. To get a list of `models` filtering by `serie` and `equipmentType` it's necessary inform the `models id` by `links.models` and `equipmentType id` by `links.equipmentType` HTTP parameter on the request.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/serie/series

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 3]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
...
+
+<blockquote class='lang-specific curl'><p>curl example to get a list of equipment types</p></blockquote>
+```curl
+curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
+       "https://agco-fuse-trackers-sandbox.herokuapp.com/equipmentTypes"
+```
+
+To retrieve a list of equipment types it's necessary to make a HTTP request to `equipment type` resource on Telemetry API.
+
+### Step 4: Get a list of models linked to a serie and equipment type
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/serie/series

File: source/includes/use_case_scenarios/_use_case_scenarios.md (hunk 2 above)
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: "it's necessary inform the `models id` by `links.models` [...]  HTTP parameter on the request".
On the curl example above there's no `links.models` HTTP parameter, is this intended?

Comment (waldemarnt) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: A list of models linked to a serie, series are the plural of serie. Make sense?

File: source/includes/use_case_scenarios/_use_case_scenarios.md (hunk 3 above)
Comment (waldemarnt) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: A list of models linked to a serie, series are the plural of serie. Make sense?

**Comment by lucaslago** [comment]
👍

**Comment by andressa** [comment]
👍

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 4]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
 
 More details can be found on link [Get single equipment section](#get-equipment-id)
 
+## Listing models filtering by series, types and brands
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Would it be possible extracting this as a separated file?

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 5]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
...
+
+To list models applying a filter by series, types and brands it is necessary to make HTTP requests to [Telemetry API](#telemetry-api) resources using specific query parameters.
+
+### Step 1: Get list of brands
+
+<blockquote class='lang-specific curl'><p>curl example to get a list of brands:</p></blockquote>
+```curl
+curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Should we just reference the `brands` endpoint documentation?

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 6]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
...
+Brands are the higher level of the hierarchy.
+
+To retrieve a list of brands it's necessary to make a HTTP request to `brands` resource on Telemetry API like the example in the right panel.
+
+### Step 2: Get a list of series linked to a brand
+
+<blockquote class='lang-specific curl'><p>curl example to get a list of series</p></blockquote>
+```curl
+curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
+       "https://agco-fuse-trackers-sandbox.herokuapp.com/series?\
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Should we just reference the `series` endpoint documentation?

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 7]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
...
+To retrieve a list of series it's necessary to make a HTTP request to `series` resource on Telemetry API.
+
+Series are linked with brands. To get a list of `series` filtering by `brand` it's necessary inform the `brand id` by `links.brand` HTTP parameter on the request.
+
+### Step 3: Get a list of equipment types
+
+<blockquote class='lang-specific curl'><p>curl example to get a list of equipment types</p></blockquote>
+```curl
+curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
+       "https://agco-fuse-trackers-sandbox.herokuapp.com/equipmentTypes"
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Should we just reference the `equipmentTypes` endpoint documentation?

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 8]
```diff
@@ -35,11 +35,64 @@ The equipment on the response contains the identification information, as the ma
...
+```
+
+To retrieve a list of equipment types it's necessary to make a HTTP request to `equipment type` resource on Telemetry API.
+
+### Step 4: Get a list of models linked to a series and equipment type
+
+<blockquote class='lang-specific curl'><p>curl example to get a list of models</p></blockquote>
+```curl
+curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Should we just reference the `series` endpoint documentation?

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 9]
```diff
@@ -35,56 +35,34 @@ More details can be found on link [Get single equipment section](#get-equipment-
...
-
-<blockquote class='lang-specific curl'><p>curl example to get a list of series</p></blockquote>
-```curl
-curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
-       "https://agco-fuse-trackers-sandbox.herokuapp.com/series?\
-links.brand=fbbdb6a9-1806-4ef1-9d4f-7841a472220e"
-```
-
-To retrieve a list of series it's necessary to make a HTTP request to `series` resource on Telemetry API.
+Using [Telemetry API](#telemetry-api) is possible to filter data as needed. A common use case is to list models
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/is possible/it is possible/

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 10]
```diff
@@ -35,56 +35,34 @@ More details can be found on link [Get single equipment section](#get-equipment-
...
+filtering by brands and series. The API provides a rich way to navigate among links and filter data using them.
 
-Series are linked with brands. To get a list of `series` filtering by `brand` it's necessary inform the `brand id` by `links.brand` HTTP parameter on the request.
+All the requests trough the AGCO APIs use the [OAuth authentication](#authentication), which is a good step to start
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/All the requests/All requests/

Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/trough the/to/

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 11]
```diff
@@ -35,56 +35,34 @@ More details can be found on link [Get single equipment section](#get-equipment-
...
-```curl
-curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
-       "https://agco-fuse-trackers-sandbox.herokuapp.com/equipmentTypes"
-```
+The following request flow shows how to list `equipment models` filtering by their `links` that are
+`brands`, `series` and `equipment types`. All the API resources are linked to their documentation
+where shows how to make requests and available parameters.
 
-To retrieve a list of equipment types it's necessary to make a HTTP request to `equipment type` resource on Telemetry API.
+1. The first step has to get a list of [brands](#get-brands) resource on `Telemetry API`.
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/has/is/

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 12]
```diff
@@ -35,56 +35,34 @@ More details can be found on link [Get single equipment section](#get-equipment-
...
-### Step 4: Get a list of models linked to a series and equipment type
-
-<blockquote class='lang-specific curl'><p>curl example to get a list of models</p></blockquote>
-```curl
-curl -X GET -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
-  "https://agco-fuse-trackers-sandbox.herokuapp.com/models?\
-  links.series=fbbdb6a9-1806-4ef1-9d4f-7841a472220e&\
-  links.equipmentType=fbbdb6a9-1806-4ef1-9d4f-7841a472220e"
-```
+2. The next request will filter [series](#series-request-parameters) linking with a `brand` using the
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/linking/linking them/

**Pull request merged**